[2023-12-12T15:44:14.809+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: testtttttt.insert_sql scheduled__2023-12-11T15:44:11.539295+00:00 [queued]>
[2023-12-12T15:44:14.817+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: testtttttt.insert_sql scheduled__2023-12-11T15:44:11.539295+00:00 [queued]>
[2023-12-12T15:44:14.818+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-12T15:44:14.828+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_sql> on 2023-12-11 15:44:11.539295+00:00
[2023-12-12T15:44:14.835+0000] {standard_task_runner.py:57} INFO - Started process 173 to run task
[2023-12-12T15:44:14.838+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'testtttttt', 'insert_sql', 'scheduled__2023-12-11T15:44:11.539295+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/testtttttt.py', '--cfg-path', '/tmp/tmpjmy7gf3k']
[2023-12-12T15:44:14.840+0000] {standard_task_runner.py:85} INFO - Job 123: Subtask insert_sql
[2023-12-12T15:44:14.886+0000] {task_command.py:416} INFO - Running <TaskInstance: testtttttt.insert_sql scheduled__2023-12-11T15:44:11.539295+00:00 [running]> on host 7fd8dd875aa1
[2023-12-12T15:44:14.950+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='xin' AIRFLOW_CTX_DAG_ID='testtttttt' AIRFLOW_CTX_TASK_ID='insert_sql' AIRFLOW_CTX_EXECUTION_DATE='2023-12-11T15:44:11.539295+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-11T15:44:11.539295+00:00'
[2023-12-12T15:44:14.952+0000] {sql.py:274} INFO - Executing: 
            COPY cctv_data(location, area, longitude, latitude)
            FROM 'D:/Airflow-docker/dags/data/CCTV_02.csv' DELIMITER ',' CSV HEADER;
            
[2023-12-12T15:44:14.959+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-12T15:44:14.968+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-12T15:44:15.011+0000] {sql.py:418} INFO - Running statement: 
            COPY cctv_data(location, area, longitude, latitude)
            FROM 'D:/Airflow-docker/dags/data/CCTV_02.csv' DELIMITER ',' CSV HEADER;
            , parameters: None
[2023-12-12T15:44:15.014+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 392, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 423, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedFile: could not open file "D:/Airflow-docker/dags/data/CCTV_02.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.

[2023-12-12T15:44:15.023+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=testtttttt, task_id=insert_sql, execution_date=20231211T154411, start_date=20231212T154414, end_date=20231212T154415
[2023-12-12T15:44:15.034+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 123 for task insert_sql (could not open file "D:/Airflow-docker/dags/data/CCTV_02.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.
; 173)
[2023-12-12T15:44:15.050+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-12T15:44:15.066+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
